tools/test-js-with-node

```diff
@@ -169,6 +169,7 @@ EXEMPT_FILES = {
     "static/js/unread_ops.js",
     "static/js/unread_ui.js",
     "static/js/upload_widget.js",
+    "static/js/user_profile.js",
```

Comment:
Currently working on adding node tests so this is a temporary addition.
I'll squash this commit along with the test one.